src/EbmlMaster.cpp

```diff
@@ -30,9 +30,9 @@ EbmlMaster::EbmlMaster(const EbmlMaster & ElementToClone)
 {
   SetSizeInfinite(!IsFiniteSize());
   ElementList.reserve(ElementToClone.ListSize());
-  // add a clone of the list
-  for (const auto& e : ElementToClone.ElementList)
-    ElementList.push_back(e->Clone());
+  const auto el = ElementToClone.ElementList;
```

Comment:
You should keep the original comment.

Also I'd be curious if this produces better code. It's certainly less readable, and thus more error-prone.

My understanding is that using STL code would allow changing the container without changing this code. But using the `back_inserter` probably means the container needs a `push_back` somewhere. So in the end there's no gain.

Reply:
Pretty sure it doesn’t actually

Reply:
removed.